src/Microsoft.Health.SqlServer/Features/Schema/SchemaJobWorker.cs

```diff
@@ -60,11 +60,12 @@ public async Task ExecuteAsync(SchemaInformation schemaInformation, string insta
                         int? previous = schemaInformation.Current;
                         schemaInformation.Current = await schemaDataStore.UpsertInstanceSchemaInformationAsync(instanceName, schemaInformation, cancellationToken);
 
-                        // If there was a change in the schema version
-                        if (previous != schemaInformation.Current)
+                        // If there was a change in the schema version and this isn't the base schema
+                        if (schemaInformation.Current != previous && schemaInformation.Current > 0)
                         {
-                            // Fire a notification
-                            await _mediator.NotifySchemaUpgradedAsync((int)schemaInformation.Current);
+                            var isFullSchemaSnapshot = previous == 0;
```

Comment:
Thank you for fixing this! For tracking purposes: this was a bug caught after PR #92 was merged (see [this comment](https://github.com/microsoft/healthcare-shared-components/commit/6e0eb86d88a6ec261cb43f4de841039f764274cb#r43051971)).